formatTest/typeCheckedTests/expected_output/mlSyntax.4.04.0.re

```diff
@@ -0,0 +1,5 @@
+module Foo = {
+  type t = {name: string};
+};
+
+let foo = (Foo.{name}) => ();
```

Comment:
Want to add some more tests covering the various places patterns can occur? Destructuring tuples, switch statements etc.